Template/Tag/FathomTag.web.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var siteID = parameters.get('siteID');
+            var fathomURL = parameters.get('fathomURL');
+            if (siteID && fathomURL) {
+                (function () {
+                    window.fathom = window.fathom || function () {
```

Comment:
I would check if `window.fathom` exists here and bail if it does, to prevent adding the `<script>` tag to the DOM multiple times.

Reply:
Thanks this makes sense.